where-to-travel/src/main/webapp/script.js

```diff
@@ -43,6 +43,9 @@ const HOURS_ID = 'hrs';
 const MINUTES_ID = 'mnts';
 const SCROLL_ID = 'scroller';
 const DASH_ID = 'dashboard';
+const FEEDBACK_ID = 'feedback-target';
+const HOURS_MAX_SEARCH = 20;
+const MINUTE_MAX_SEARCH = 60;
```

Comment:
What do you think about making MINUTE_MAX_SEARCH 59 instead?

Reply:
Done